SportsHub/SportsHub.Application/Services/UserService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using SportsHub.Domain.Models;
+using SportsHub.Domain.Repository;
+
+namespace SportsHub.AppService.Services
+{
+    public class UserService : IUserService
+    {
+        public UserService(IUnitOfWork uow)
+        {
+
+        }
+
+        public User GetByPassword()
+        {
+            throw new NotImplementedException();
```

Comment:
use this and calls it from the controller

Reply:
Implemented